recipes-multimedia/aamp/aampmetrics_git.bb

```diff
@@ -0,0 +1,23 @@
+SUMMARY = "AAMP Metris library"
+DESCRIPTION = "Metrics Library is an independent library for session statistics generation/reporting in JSON format"
+HOMEPAGE = "https://github.com/rdkcmf/rdk-aampmetrics"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
+
+DEPENDS_append = " cjson"
+
+PV = "0.1.gitr${SRCPV}"
+
+SRC_URI = "git://github.com/rdkcmf/rdk-aampmetrics.git;branch=stable2;protocol=https"
```

Comment:
Can you please add BRANCH variable and make a weak assignment for SRCREV and BRANCH?